packages/botonic-react/src/components/element.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 import styled from 'styled-components'
-import { COLORS } from '../constants'
+import { COLORS, WEBCHAT } from '../constants'
 import { renderComponent } from '../utils'
 
 const ElementContainer = styled.div`
   display: flex;
   flex-direction: column;
-  width: 222px;
+  width: ${WEBCHAT.DEFAULTS.WIDTH_ELEMENT}px;
   margin-right: 6px;
```

Comment:
you can also add a reference for this `6px` like you did for `WEBCHAT.DEFAULTS.ELEMENT_WIDTH`. `WEBCHAT.DEFAULTS.ELEMENT_MARGIN_RIGHT` which in in fact are the ones that make arrows not completely disappearing when you click on one of boths arrows and you reached the end of a carousel element.